src/util.js

```diff
@@ -0,0 +1,46 @@
+export class util {
```

Comment:
Shouldn't be a class.This file should just contain a collection of helper methods.